lib/helper.js

```diff
@@ -225,6 +225,43 @@ class Helper {
   static isNumber(obj) {
     return typeof obj === 'number' || obj instanceof Number;
   }
+
+  /**
+   * @param {function(?)} nodeFunction
+   * @return {function(?):!Promise<?>}
+   */
+  static promisify(nodeFunction) {
```

Comment:
why not use @ljharb's wonderful https://www.npmjs.com/package/util.promisify

Reply:
@graingert no need to pull in a dependency for 30 lines of code.

Reply:
when it's reliable and faster in modern node, it's always better to have more deps :-)